releases/index.md

```diff
@@ -68,6 +68,20 @@ The following binaries are not suitable for production environments:
             {% endif %}
         {% endfor %}
 
+        {% assign v_docker_arm = "false" %}
+        {% assign v_show_docker_multi_platform_para = "false" %}
+        {% for r in releases %}
+            {% if r.docker_arm == "true" %}
+                {% assign v_docker_arm = "true" %}
+                {% assign v_show_docker_multi_platform_para = "true" %}
+                {% break %}
+            {% else %}
+              {% assign v_docker_arm = "false" %}
+              {% assign v_show_docker_multi_platform_para = "false" %}
+                {% break %}
```

Comment:
I don't think these lines are necessary. Basically, we assume via line 71 that `v_docker_arm` and `v_show_docker_multi_platform_para` are false. What this `else` statement seems to do is just re-declare the same variables and then break the entire `for` loop after the first release.

Also, `v_docker_arm` appears to be declared but not used. Can you confirm?

Reply:
You're right -- I changed this logic in the per-release page but not here. I need `v_docker_arm` for the Docker section intro para that happens outside of the `for releases` loop. Now updated.